src/index.ts

```diff
@@ -274,6 +274,42 @@ const main = async () => {
         }
       }
     )
+    .command(
+      ["getPosition"],
+      "Get position from exchange",
+      async (yargs: Argv) => {
+        return yargs
+          .option("trader", {
+            alias: "t",
+            describe: "traders account",
+            type: "string",
+            default: "0",
+          });
+      },
+      async (argv) => {
+        const { trader } = argv;
+
+        const { accountNumber, networkId, exchangeAddress } =
+          getStandardParams(argv);
+
+        const wallet = loadAccount(networkId, accountNumber);
+        const exchange = IExchange__factory.connect(exchangeAddress, wallet);
+
+        try {
+          const trade = await exchange.getPosition(trader);
+
+          console.log({
+            asset: trade.asset.toString(),
+            stable: trade.stable.toString(),
+            adlTrancheId: trade.adlTrancheId.toString(),
+            adlShareClass: trade.adlShareClass.toString(),     
```

Comment:
`adlTrancheId` and `adlShareClass` are rather internal.
Do we want to show them in this UI?
In particular, `adlShareClass`, I would think, is not very meaningful to the end users.